src/plugins/auto_batch/tests/functional/behavior/ov_plugin/auto_batching_tests.cpp

```diff
@@ -31,29 +28,4 @@ INSTANTIATE_TEST_SUITE_P(smoke_AutoBatching_test,
                                             ::testing::ValuesIn(num_requests),
                                             ::testing::ValuesIn(num_batch)),
                          AutoBatching_Test_DetectionOutput::getTestCaseName);
-
-INSTANTIATE_TEST_SUITE_P(smoke_AutoBatching_test,
-                         DefaultConfigurationTest,
-                         ::testing::Combine(::testing::Values(std::string(ov::test::utils::DEVICE_BATCH) + ":" +
-                                                              ov::test::utils::DEVICE_TEMPLATE),
-                                            ::testing::Values(DefaultParameter{CONFIG_KEY(AUTO_BATCH_TIMEOUT),
-                                                                               InferenceEngine::Parameter{"1000"}})),
-                         DefaultConfigurationTest::getTestCaseName);
-
-INSTANTIATE_TEST_SUITE_P(smoke_AutoBatching_test_string,
-                         DefaultConfigurationTest,
-                         ::testing::Combine(::testing::Values(std::string(ov::test::utils::DEVICE_BATCH) + ":" +
-                                                              ov::test::utils::DEVICE_TEMPLATE),
-                                            ::testing::Values(DefaultParameter{ov::auto_batch_timeout.name(),
-                                                                               InferenceEngine::Parameter{"1000"}})),
-                         DefaultConfigurationTest::getTestCaseName);
-
-INSTANTIATE_TEST_SUITE_P(
-    smoke_AutoBatching_test_uint,
-    DefaultConfigurationTest,
-    ::testing::Combine(
-        ::testing::Values(std::string(ov::test::utils::DEVICE_BATCH) + ":" + ov::test::utils::DEVICE_TEMPLATE),
-        ::testing::Values(DefaultParameter{ov::auto_batch_timeout.name(), InferenceEngine::Parameter{uint32_t(1000)}})),
-    DefaultConfigurationTest::getTestCaseName);
-
```

Comment:
Is there any replacement for DefaultConfigurationTest?

Reply:
I think property tests should be enough for this, am I right?